feat(curriculum): add animations to block 3.2 of the English Curriculum (#54033)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-talk-about-hobbies-and-interests/657da1d38bf3e693eb579be9.md

---
id: 657da1d38bf3e693eb579be9
title: Task 25
challengeType: 22
dashedName: task-25
---

<!-- (Audio) The whole dialog. -->

---
# --description--

> In this challenge, you will practice what you have learned in the challenges for this lesson. You will fill in the blanks with the appropriate words and the appropriate forms of verbs related to it.

Reviewer: ```suggestion
In this challenge, you will apply what you have learned from the previous challenges in this lesson. Fill in the blanks with the appropriate words and the correct forms of the verbs related to them.
```

Author: Great catch, Sem! Thank you!
